src/ui/zcl_abapgit_gui_page_addofflin.clas.abap

```diff
@@ -27,14 +27,14 @@ CLASS zcl_abapgit_gui_page_addofflin DEFINITION
         url            TYPE string VALUE 'url',
         package        TYPE string VALUE 'package',
         folder_logic   TYPE string VALUE 'folder_logic',
-        labels         TYPE string VALUE 'labels',
```

Comment:
can we keep all `ids` on the same page instead of having them in multiple places?